### PR TITLE
Adding async/await example to documentation

### PR DESCRIPTION
**Pull request by mrgcohen**
Only additions to the documentation.

### CODE REVIEW

**Comment by sushantdhiman** [comment]
Linting fails

```
docs/migrations.md: 296: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs/migrations.md: 297: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs/migrations.md: 300: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs/migrations.md: 304: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs/migrations.md: 305: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs/migrations.md: 307: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs/migrations.md: 309: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs/migrations.md: 311: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 12]
docs/migrations.md: 323: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs/migrations.md: 324: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs/migrations.md: 327: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs/migrations.md: 329: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 7]
```

File: docs/migrations.md [hunk 1]
```diff
@@ -285,6 +285,57 @@ module.exports = {
 
 ```
 
+The next is an example of a migration that has uses async/await where you create an unique index on a new column:
+
+```js
+module.exports = {
+  up: async(queryInterface, Sequelize) => {
```
Comment (papb) [inline docs/migrations.md]: May I suggest the shorthand syntax to be used instead?

```suggestion
async up(queryInterface, Sequelize) {
```

File: docs/migrations.md [hunk 2]
```diff
@@ -285,6 +285,57 @@ module.exports = {
...
+         { transaction }
+       );            
+       await transaction.commit();
+    } catch (err) {
+      await transaction.rollback();
+      throw err;
+    }
+  },
+
+  down: (queryInterface, Sequelize) => {
```
Comment (papb) [inline docs/migrations.md]: Here you forgot the `async`

```suggestion
async down(queryInterface, Sequelize) {
```

Comment (mrgcohen) [reply docs/migrations.md]: thanks!

**Comment by mrgcohen** [comment]
Ok, linting is hopefully fixed. Sorry typed it quick in the github editor.

**Pull request merged**